code/src/index.css

```diff
@@ -1,13 +1,246 @@
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	font-family: 'Fanwood Text', serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.main-container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background-image: url(assets/start.jpg);
+	height: 100vh;
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-position-y: 1%;
+	background-position-x: 50%;
+	position: relative;
+}
+
+.start-container {
+	background-color: #d8bf978a;
+	border-radius: 25px;
+	display: flex;
+	flex-direction: column;
+	padding: 11px;
+	width: 85%;
+	color: #292b1d;
+	align-items: center;
+}
+
+.start-container h1 {
+	font-family: 'Ingrid Darling';
+	color: #292b1d;
+	font-size: 50px;
+	margin: 0;
+}
+
+.start-container p {
+	font-family: 'Fanwood Text', serif;
+	color: #292b1d;
+	font-size: 18px;
+	text-align: center;
+	margin: 0;
+}
+
+.start-input {
+	font-family: 'Fanwood Text', serif;
+	font-size: 18px;
+	border-radius: 5px;
+	padding: 5px 2px;
+	text-align: center;
+	border: solid #292b1d 2px;
+	color: black;
+	background-color: whitesmoke;
+}
+
+.start-button {
+	font-family: 'Ingrid Darling';
+	font-weight: bold;
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+	padding: 12px;
+	border-radius: 5px;
+	border: solid #292b1d 2px;
+	color: #292b1d;
+	background-color: #868d65e0;
+	font-size: 22px;
+	letter-spacing: 1px;
+	transition: all 0.15s linear;
+	text-transform: uppercase;
+}
+
+.start-button:hover,
+button:hover {
+	background-color: rgba(29, 201, 160, 0.08);
+	border-color: #2a382f;
+	color: #2a382f;
+	transform: translateY(-5px) scale(1.05);
+	text-transform: uppercase;
+}
+
+.start-button:active,
+button:active {
+	background-color: transparent;
+	border-color: #2a382f;
+	color: #2a382f;
+	transform: translateY(5px) scale(1);
+	text-transform: uppercase;
+}
+
+button {
+	font-family: 'Fanwood Text', serif;
+	font-weight: bold;
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+	padding: 12px;
+	border-radius: 25px;
+	border: solid #292b1d 2px;
+	color: #292b1d;
+	background-color: #868d65e0;
+	font-size: 12px;
+	letter-spacing: 1px;
+	transition: all 0.15s linear;
+	text-transform: uppercase;
+	margin: 10px 0px 10px;
+}
+
+button:disabled {
+	background-color: rgba(255, 255, 255, 0.16);
+	color: #8e8e93;
+	border-color: #8e8e93;
+	margin: 0;
+}
+
+.back-button {
+	border-radius: 5px;
+	padding: 5px 2px;
+	color: #292b1d;
+	border: solid #292b1d 2px;
+	background-color: #868d65c7;
+	font-family: 'Fanwood Text', serif;
+	font-size: 12px;
+	font-weight: bold;
+	position: absolute;
+	top: 15px;
+	left: 15px;
+}
```

Comment:
This button is could be moved to be placed closer to the other text content in desktop. For us it was a bit hard to find.